python_bindings/carcassonne_engine/requests.py

```diff
@@ -13,6 +13,8 @@
     "MoveWithState",
 )
 
+from .placed_tile import PlacedTile
```

Comment:
Put this alongside the rest of the imports (i.e. above `__all__`).

Reply:
moved